articles/active-directory/manage-apps/tenant-restrictions.md

```diff
@@ -101,6 +101,9 @@ While configuration of tenant restrictions is done on the corporate proxy infras
 
 The admin for the tenant specified as the Restricted-Access-Context tenant can use this report to see sign-ins blocked because of the tenant restrictions policy, including the identity used and the target directory ID. Sign-ins are included if the tenant setting the restriction is either the user tenant or resource tenant for the sign-in.
 
+> [!NOTE]
+> The report may contain limited information such as target directory ID when users in tenant other than specified as the Restricted-Access-Context tenant sign in. In this case, user identity such as name and user principal name are masked to protect user data in other tenants.
```

Comment:
Thanks for the contribution - please look at these edits to make sure the info is still accurate:
```suggestion
> The report may contain limited information, such as target directory ID, when a user who is in a tenant other than the Restricted-Access-Context tenant signs in. In this case, user identifiable information, such as name and user principal name, is masked to protect user data in other tenants.
```